pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -1494,13 +1565,16 @@ func listRegistrationEntriesOnce(ctx context.Context, db *sqlDB, req *datastore.
 	return resp, nil
 }
 
-func buildListRegistrationEntriesQuery(dbType string, req *datastore.ListRegistrationEntriesRequest) (string, []interface{}, error) {
+func buildListRegistrationEntriesQuery(dbType string, supportsCTE bool, req *datastore.ListRegistrationEntriesRequest) (string, []interface{}, error) {
 	switch dbType {
 	case SQLite:
 		return buildListRegistrationEntriesQuerySQLite3(req)
 	case PostgreSQL:
 		return buildListRegistrationEntriesQueryPostgreSQL(req)
```

Comment:
Oh, I guess here too - same comment as above.

Seeing this occur a second time makes me lean towards the "unconditionally accept supportsCTE parameter" option, but whatever you prefer that addresses the original concern is OK with me

Reply:
I now understand what you mean.... i'll make it clear.